test/NuGet.Clients.Tests/NuGet.Credentials.Test/PluginManagerMock.cs

```diff
@@ -210,6 +210,9 @@ private void EnsureAllEnvironmentVariablesAreCalled(string pluginFilePath)
             _reader.Setup(x => x.GetEnvironmentVariable(
                     It.Is<string>(value => value == CredentialTestConstants.PluginHandshakeTimeoutEnvironmentVariable)))
                         .Returns("HandshakeTimeout");
+            _reader.Setup(x => x.GetEnvironmentVariable(
```

Comment:
can you add a test where this env is disabled and test we dont initialize download plugin?